src/cargo/ops/cargo_compile.rs

```diff
@@ -423,11 +423,12 @@ pub fn compile_ws<'a>(
             &resolve_with_overrides,
             &packages,
             config,
-            build_config,
+            &build_config,
             profiles,
             extra_compiler_args,
         )?;
-        cx.compile(&units, export_dir.clone(), &exec)?
+        let mut progress = BuildProgress::new(config, &cx)?;
```

Comment:
Note that we can remove `mut` from `cx` now! Not sure why `rustc` does not warn about it though...

Reply:
Nice catch. I'll fix it, and I've written up a rustc bug: https://github.com/rust-lang/rust/issues/50321.